das/local_disk_das.go

```diff
@@ -87,11 +88,23 @@ func NewLocalDiskDataAvailabilityService(dbPath string, signerMask uint64) (*Loc
 		}
 	}
 
+	keyset := &arbstate.DataAvailabilityKeyset{
+		AssumedHonest: 1,
+		PubKeys:       []blsSignatures.PublicKey{*pubKey},
+	}
```

Comment:
If the idea is that the individual DAS servers that are part of the total set are running in local mode, is this an issue?